modules/cassandra/src/main/java/org/apache/ignite/cache/store/cassandra/utils/session/CassandraSessionImpl.java

```diff
@@ -221,7 +221,7 @@ else if (CassandraHelper.isPreparedStatementClusterError(e))
                 if (!tblAbsenceErrorFlag && !hostsAvailabilityErrorFlag && !prepStatementErrorFlag)
                     throw new IgniteException(errorMsg, error);
 
-                if (log != null && !CassandraHelper.isPreparedStatementClusterError(error))
+                if (!CassandraHelper.isPreparedStatementClusterError(error))
```

Comment:
It will produce NullPointerException if Ignite was started with Basic configuration without Spring support added (see the same comment above)